scripts/game_structure/windows.py

```diff
@@ -462,14 +462,14 @@ class SpecifyCatGender(UIWindow):
     """This window allows the user to specify the cat's gender"""
 
     def __init__(self, cat):
-        super().__init__(scale(pygame.Rect((600, 430), (800, 370))),
+        super().__init__(scale(pygame.Rect((560, 360), (845, 470))),
```

Comment:
Regarding UI - I think this UI is too simple, and not very user friendly. There are no labels on the pronoun entry boxes, which may be confusing. In addition, there is no way to select from default pronouns, nor a way to add multiple pronouns.

Reply:
We got it so you can select default pronouns, custom pronouns, have multiple pronouns. There's labels on the pronoun entry boxes. And an example box so you can test pronouns.